src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -130,6 +135,72 @@ public void cachesGetRequest() throws Exception {
     }
 
     /**
+     * LastModifiedCachingWire can evict any previous cached entry if a new
+     * response does not have a last modified header.
+     * We can observe this via the If-Modified-Since headers as when the cache
+     * does not contain an entry, this is not present on the request.
+     * @throws Exception If fails
+     */
+    @Test
+    public void doesNotCacheGetRequestIfTheLastModifiedHeaderIsMissing()
+        throws Exception {
+        final String firstresponse = "Body 1";
+        final String secondresponse = "Body 2";
+        final String thirdreponse = "Body 3";
+        final Map<String, String> headers = Collections.singletonMap(
```

Comment:
@mkordas again, just copying the original style of the tests, so will raise another todo around them